### ISSUE TITLE
:Dirbuf works only until restart, then opens netrw

### ISSUE THREAD

**Issue by JSchrtke** [elihunter173/dirbuf.nvim#9]
# Expected
`:Dirbuf` to open dir buffer every time

# Actual
using `:Dirbuf` only opens the dir buffer if used directly after installing. Closing and reopening neovim after causes the `:Dirbuf` command to open netrw instead.

# Repro
* Install `dirbuf` plugin
* `:Dirbuf`
* `:q`
* `nvim`
* `:Dirbuf`

this will open netrw

# Minimal repro config
```vim
call plug#begin('~/.vim/plugged')

Plug 'elihunter173/dirbuf.nvim'

call plug#end()
```

```
NVIM v0.6.1
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by root@js-ubuntu

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/home/js/local/nvim/share/nvim"

Run :checkhealth for more info
```

```
Ubuntu 20.04.3 LTS x86_64
```

Also confirmed on Windows 10, same config and repro steps.

**Comment by elihunter173** [comment]
Thank you for the detailed bug report! I might steal this to make an issue template.

I managed to reproduce the issue. I thought I fixed this issue a couple months ago but alas this is what I get for disabling Netrw in my config.

**Comment by JSchrtke** [comment]
> Thank you for the detailed bug report!
Of course, least I can do.

> ... I get for disabling Netrw in my config.
Mind sharing that as a workaround? I'm itching to try this thing out at work tomorrow ;)

**Comment by elihunter173** [comment]
Here's the workaround! ```lua
vim.g.loaded_netrwPlugin = 1
vim.g.loaded_netrw = 1
```

Though I should have the bugfix ready already. I believe the issue is I was hijacking Netrw's file handler when the plugin is loaded rather than on the `VimEnter` autocommand. I'm just double checking to make sure nothing is broken and then I'll push the fix

**Comment by elihunter173** [comment]
Fixed in 733fef744c229300c9e30e511854bb637910b328. Let me know if you catch any more bugs!

**Comment by JSchrtke** [comment]
Cool, even better. Seems to be working at first glance (on Linux/Ubuntu). I'll let you know should I encounter more issues tomorrow.
